website/docs/getting-started/04--contribution.md

```diff
@@ -1,8 +1,22 @@
 ---
 title: Contribution
-description: This is the (missing) long description of the component, that will come from the frontmatter attributes
+description: Contributing to the HashiCorp Design System
```

Comment:
I don't think this page needs a description; it doesn't feel like it's providing additional value.

```suggestion
```